gluecodium/src/main/resources/templates/dart/DartFunctionDocs.mustache

```diff
@@ -18,23 +18,8 @@
   ! License-Filename: LICENSE
   !
   !}}
-{{#resolveName comment}}{{#unless this.isEmpty}}{{prefix this "/// "}}
```

Comment:
I have to split this the doc part and attribute part. Otherwise, the lambda gets some duplicated attributes [here](https://github.com/heremaps/gluecodium/blob/46a30559cade06edd7bf56c7053c776061bda071/gluecodium/src/test/resources/smoke/attributes/output/dart/lib/src/smoke/attributes_lambda.dart#L4)